src/main/java/seedu/nutrigoals/ui/HelpWindow.java

```diff
@@ -90,13 +124,36 @@ public void focus() {
     }
 
     /**
-     * Copies the URL to the user guide to the clipboard.
+     * Returns all the commands and its usage
+     * @return A treemap containing all the commands and its usage
      */
+    public TreeMap<String, String> getCommands() {
+        String regex = ": ";
+        TreeMap<String, String> treeMap = new TreeMap<>();
+        treeMap.put(AddCommand.COMMAND_WORD, AddCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(ClearCommand.COMMAND_WORD, ClearCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(DeleteCommand.COMMAND_WORD, DeleteCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(EditCommand.COMMAND_WORD, EditCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(ExitCommand.COMMAND_WORD, ExitCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(FindCommand.COMMAND_WORD, FindCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(HelpCommand.COMMAND_WORD, HelpCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(ListCommand.COMMAND_WORD, ListCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(LocateGymCommand.COMMAND_WORD, LocateGymCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(ProfileCommand.COMMAND_WORD, ProfileCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(ReviewCommand.COMMAND_WORD, ReviewCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(SetupCommand.COMMAND_WORD, SetupCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(SuggestCommand.COMMAND_WORD, SuggestCommand.MESSAGE_USAGE.split(regex, 2)[1]);
+        treeMap.put(TargetCommand.COMMAND_WORD, TargetCommand.MESSAGE_USAGE.split(regex, 2)[1]);
```

Comment:
I think the message usage for `TargetCommand` currently just states `Parameters: Integer`. Since users are not allowed to key in a calorie target that is too large, would it be clearer to change it to something like `Parameters: Integer (must not be too large)`?